neuralmonkey/trainers/lr_decay.py

```diff
@@ -0,0 +1,38 @@
+"""Collection of learning rate decay strategies."""
```

Comment:
Zkusil bych se zamyslet nad tím, jestli by to nešlo pojmenovat nějak obecněji - ve světle toho článku, co doporučuje ten learning rate sinusovkovat.

Reply:
Tak obecne muzeme zavest learning_rate_modifier (nebo decay) funkce, ktere budou dostavat lr a step a budou vracet v case upraveny lr.

Reply:
Nejsem proti mit vsechny tyto funkce mit nahazene v jednom zdrojaku (treba ten neuralmonkey.functions). A v budoucnu az by jich bylo hodne, tak by se to treba rozhazelo do vice zdrojaku.